main/icepage.js

```diff
@@ -619,83 +619,87 @@ function isScrolledIntoView(elem)
 }
 
 function makeTagMenu(includeTags) {
```

Comment:
On some older computers I have anything jQuery runs a bit choppy. We may not notice it but it will still affect some users.

In response to your comment on Discord, I believe that people should learn basic JS first before jQuery. It is important to know how you are doing something and not just that you are doing it.